Check multicall section validity for cairo1 (#1930)
software-mansion/protostar: website/docs/cairo-1/06-interacting-with-starknet/06-multicall.md

inputs = [42]
```

> Protostar supports two types of transactions — `deploy` and `invoke`, which take similar arguments to [`deploy` command](/docs/cli-reference#deploy) and [`invoke` command](/docs/cli-reference#invoke).

Reviewer: ```suggestion
Protostar supports few types of transactions, including `deploy` and `invoke`, which take similar arguments to [`deploy` command](/docs/cli-reference#deploy) and [`invoke` command](/docs/cli-reference#invoke).
```

I think we shouldn't be saying that 2 transactions are supported if we also support declare and deploy_account.

Author: I rephrased it so it might be more clear now

> When choosing `deploy` transaction type, the deployment happens through the Universal Deployer Contract (UDC), see the [`deploy` command](/docs/cli-reference#deploy) for more information.

Reviewer: ```suggestion
With the `deploy` transaction type, the deployment happens through the Universal Deployer Contract (UDC), see the [`deploy` command](/docs/cli-reference#deploy) for more information.
```

Also we could consider linking to this https://docs.openzeppelin.com/contracts-cairo/0.6.1/udc
